heidrun/pa_dc.rb

```diff
@@ -0,0 +1,108 @@
+
+# Convert ValueArray to an array to manipulate and then reinstantiate a new ValueArray
+#   If more than one dc:contributor property exists then it drops the last one 
+#   If one or zero exist then do nothing
+# 
+# Feels kludgey as heck, should rewrite once ValueArray supports array sliceing 
+# See ticket https://issues.dp.la/issues/8287
+contributor_top = lambda do |r|
+  # testing 
+  # r['dc:contributor'].first_value( r['dc:contributor'].count-1 )
+  # end test
+
+  if r['dc:contributor'].count > 1 
+    r['dc:contributor'].first_value( r['dc:contributor'].count-1 )
+  else 
+    r['dc:contributor']
+  end 
+end
+
+# dc:relation 
+# If there is more than one dc:relation property than return all but the first 
+# instance of the property. If one or none then return an empty set
+#
+secondary_relations = lambda do |r|
+  if(r['dc:relation'].count > 1)
+    r['dc:relation'].last_value( r['dc:relation'].count-1 )
+  else 
+    # TODO there is probably a better way return an empty set that isn't nil
```

Comment:
I'm seeing you need this to `#flatten`.  I'm wondering if just `[]` doesn't work?